spec/features/search/search_spec.rb

```diff
@@ -0,0 +1,70 @@
+require 'rails_helper'
+
+RSpec.describe 'search results page', type: :feature do
+  describe 'As a visitor' do
+    describe 'When I visit the welcome page' do
+
+      it 'I can fill in a word, click search, I am taken to the /quotes page and I see the total number of quotes that were found as well as the keyword that is being used for the search.' do
+
+        visit root_path
+        fill_in :keyword, with: 'I am'
+        click_on "Search for Quotes"
+        expect(page).to have_content('Total Number of Quotes: 151')
+        expect(page).to have_content('Keyword Used for this Search: I am')
+      end
+
+      it 'I also see a list of the top 10 quotes that were found. Under each quote I see the author & category(ies) or tags' do
+        visit root_path
+        fill_in :keyword, with: 'I am'
+        click_on "Search for Quotes"
+
+        expect(page).to have_content('Top 10 Quotes Found')
+        within("#quote_#{0}") do 
+          expect(page).to have_content("Rabbi Hillel") # I don't know how to make this dynamic at the moment
+          expect(page).to have_content("If I am not for myself, who will be for me? If I am not for others, what am I? And if not now, when?")
+          expect(page).to have_content("famous-quotes")
+          expect(page).to have_content("wisdom")
+        end
+      end
```

Comment:
You could get around hardcoding by using the id that you're storing in the quote poro. Let's say in your view, your id for each quote on the page was ".quote-#{quote.id}". In your test, you could do something like: 

```ruby
total_count, quotes = QuoteFacade.get_quotes_info("I am") 
quotes.each do |quote|
      within ".quote-#{quote.id}" do 
        expect(page).to have_content(quote.content)
        expect(page).to have_content(quote.author)
        quote.tags.each do |tag|
          expect(page).to have_content(tag)
        end 
      end 
```